scans/worker.py

```diff
@@ -0,0 +1,41 @@
+import os
+import json
+import boto3
+dynamodb = boto3.resource('dynamodb')
+table = dynamodb.Table(os.environ['DYNAMODB_TABLE'])
+
+
+def worker(event, context):
+    for record in event["Records"]:
+        if not record["eventName"] == "INSERT":
+            continue
+
+        item = record["dynamodb"]["NewImage"]
+        scan_id = item["id"]["S"]
+        target = item["target"]["S"]
+        port = item["target"]["I"]
+
+        # Use this to give us a signal it's working
+        logger.info('{0} {1}:{2}'.format(
+            scan_id, target, port))
+
+        # TODO: Add real scan logic here to populate scan var, probably via a scan-engine
+        #  class implemented in Python instead of Ruby
```

Comment:
I can probably get away with just running the ruby engine in a docker container as part of the worker class here, just to save from having to re-invent this.